README.md

```diff
@@ -55,7 +55,7 @@ the ruleset.
 ## Projects using rules_xcodeproj
 
 - Cash App
-- [Envoy Mobile](https://github.com/envoyproxy/envoy-mobile)
+- [Envoy Mobile](https://github.com/envoyproxy/envoy/tree/main/mobile)
```

Comment:
Now that the source config is linked directly in the external examples, maybe this should link to the project's online home rather than the source directory in GitHub?

```suggestion
- [Envoy Mobile](https://envoymobile.io)
```

Reply:
Yeah that's fine

Reply:
Then you should probably also remove the SwiftLint link.

However, then this list becomes a less useful reference because the examples links is in another doc.

Maybe you combine the lists here?

I see presentations, blog posts, tweets and examples as being in the same category of reference materials.